aea/cli/fingerprint.py

```diff
@@ -189,3 +189,17 @@ def fingerprint_package(
     # Load item specification yaml file and add fingerprints
     config.fingerprint = fingerprints_dict
     config_loader.dump(config, open_file(config_file_path, "w"))
+
+
+@check_aea_project(check_finger_prints=False)  # pylint: disable=no-value-for-parameter
+def fingerprint_agent(click_context: click.Context) -> None:
+    """Do a fingerprint for an agent."""
+    ctx = cast(Context, click_context.obj)
+    fingerprints_dict = _compute_fingerprint(
+        Path(ctx.cwd),
+        ignore_patterns=ctx.agent_config.fingerprint_ignore_patterns,
+        is_recursive=False,
+    )  # type: Dict[str, str]
+    ctx.agent_config.fingerprint = fingerprints_dict
+    ctx.dump_agent_config()
+    click.echo(f"Fingerprint for agent `{ctx.agent_config.name}` is calculated!")
```

Comment:
To be consistent with other commands, move to line 198 and print 
`Fingerprinting files in agent project 'agent_name' ...`